test/support.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace Chai {
+  interface Assertion {
+    deepEqual(expected: any): Assertion;
```

Comment:
Where would we need this for?

Reply:
`support.js` declares a custom chai method:

```
chai.Assertion.addMethod('deepEqual', function (expected, depth = 5) {
  expect(inspect(this._obj, { depth })).to.deep.equal(inspect(expected, { depth }));
});
```

which works like `deep.equal` but includes symbols

This declares it to TypeScript so `expect(y).to.deepEqual(x)` is valid

Reply:
Ah, that makes sense